### DISCUSSION TITLE
What is the best way to handle deleting records in a join table when primary item is deleted?

### DISCUSSION THREAD

**Discussion by andyjamesn** [supabase/supabase#7678] [Questions] [answered]
Say I have the following DB structure

**cars**
id(uuid), name

**features**
id(uuid), name

**car_features (Join table)**
id(uuid), car_id(fk_cars), feature_id(fk_features)

What is the best way to delete a car that has several features linked in the car_features table at the same time?

I know if it is a one to one relationship with a reference in the primary table to another table I could use `delete cascade` but in this case the primary table does not have a reference to the join table.

**Comment by burggraf** [comment, marked as answer]
You could use a PostgreSQL trigger for that.
[Jon Meyers has a video on triggers](https://www.youtube.com/watch?v=0N6M5BBe9AE&ab_channel=Supabase)
